service/src/test/java/bio/terra/workspace/service/workspace/WorkspaceServiceTest.java

```diff
@@ -54,7 +56,9 @@
 import org.mockito.Mockito;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.test.context.ActiveProfiles;
 
+@ActiveProfiles({"azure", "azure-test"})
```

Comment:
I am unsure about adding these profiles to this test, would love thoughts on this one (factor into another azure specific workspace service test?)

Reply:
Good question. Since azure is not enabled in all environments, I would suggest continuing to segregate the azure tests. You can make derive your test class from `BaseAzureTest` and not worry about setting the profiles.